### DISCUSSION TITLE
labbed_musical_chime playing at random times

### DISCUSSION THREAD

**Discussion by ssuppe** [NightscoutFoundation/xDrip#2989] [Q&A] [answered]
Hi all,

I'm having an issue where I am getting seemingly random audio alerts from xDrip. I've inspected the APK and found that the sound that is playing is labbed_musical_chime.ogg. I've also looked through the code but haven't yet been able to figure out what is triggering this sound, as it's used in multiple places in the code. It's not coinciding with any notification I can tell, nor is there any obvious event in the logs.

I'm using xDrip+ version 23ccbd1-2023.07.16 on a Pixel 5a, also using the watchface on my Pixel watch. I use xDrip for CGM with a Dexcom G6, although I've recently started using AndroidAPS in 'open loop' mode with a virtual pump. (I have a Medtronic 640G currently which is not supported by AndroidAPS, but will switch soon).

It's worth noting I used to use the Libre Freestyle 2, and didn't have this problem. It only emerged last week when I switched to Dexcom G6.

AFAIK I am not using Android Automotive features, although I do have Android Automotive in my car, and I do plug my phone in, so it's possible I've triggered something there. I so far haven't experienced the sound while in the car.

I realize this may be difficult to debug, but I'd appreciate any pointers.

Thank you,
Steve

**Comment by psonnera** [comment, marked as answer]
Any chance it's profile import sounds? https://xdrip.readthedocs.io/en/latest/use/interapp/#receiving-from-nsclient

**Reply by ssuppe** [reply]
Oh, interesting! I missed this. It WAS enabled. I've just enabled it and I'll see if this stops it happening, and then reply here. Thank you!

**Reply by ssuppe** [reply]
Just following up - it's been ~24 hours and I've not received any more notifications after disabling the profile import sounds. I'll go ahead and mark above as the answer and close the discussion. Thank you for the help!
